### ISSUE TITLE
Ansatz library brainstorming

### ISSUE THREAD

**Issue by gatorwatt** [tensorflow/quantum#564]
The impression I got from tutorials was that in general, other than for random gates, the convention is that quantum circuit ansatz are constructed manually. (If I am mistaken about this point then please feel free to disregard.) Would it be possible to assemble a library of common ansatz architecture initialization functions that can be initialized in simpler fashion by just designating a few points like qubit count and etc? Please consider this brainstorming.

**Comment by gatorwatt** [comment]
Yeah is not clear to me how varied are types of common ansatz. The tensorflow quantum paper demonstrates a few, wasn’t sure if these forms are like authoritative accepted mainstream architectures or just starting points.

**Comment by mhucka** [comment]
Thank you for the idea. Due to limited resources, I regret I have to close this as not planned. If that changes in the future, we can reopen this issue.
